partner_compassion/__openerp__.py

```diff
@@ -45,7 +45,7 @@
     'author': 'Compassion CH',
     'website': 'http://www.compassion.ch',
     'depends': ['base', 'partner_firstname', 'base_location',
-                'email_template'],
+                'email_template', 'contract_compassion'],
```

Comment:
Why did you add dependency to module contract_compassion ? I think it is a bad idea because it can add a cyclic dependency...